src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import mate.academy.spring.model.dto.CinemaHallRequestDto;
+import mate.academy.spring.model.dto.CinemaHallResponseDto;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.mapper.CinemaHallMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallMapper cinemaHallMapper;
+    private final CinemaHallService cinemaHallService;
+
+    public CinemaHallController(CinemaHallMapper cinemaHallMapper,
+                                CinemaHallService cinemaHallService) {
+        this.cinemaHallMapper = cinemaHallMapper;
+        this.cinemaHallService = cinemaHallService;
+    }
+
+    @GetMapping
+    public List<CinemaHallResponseDto> getAllCinemaHalls() {
+        return cinemaHallMapper.getAllCinemaHalls();
+    }
+
+    @PostMapping
+    public CinemaHallResponseDto addCinemaHall(
```

Comment:
```suggestion
    public CinemaHallResponseDto add(
```
check it everywhere 
<img width="1055" alt="image" src="https://user-images.githubusercontent.com/96740815/208452206-f0e484fd-6dda-43ea-bb9c-0dba3858ea7d.png">